### DISCUSSION TITLE
Как организовать поиск по базе данных

### DISCUSSION THREAD

**Discussion by dmbrtlt** [totumonline/totum-ru-issues-and-discussions#603] [Как это сделать?] [answered]
Допустим, есть база данных контрагентов, как отфильтровать пользователя (не передавая всех контрагентов в поле селект префильтра). Если оставлять поле строка, то ищет только строгое соответствие, а не по совпадающей части введенного слова.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Префильтры работают только по полному соответствию.

Если вы в префильтре делаете поле Селект для поиска контрагента, то в Селект не передаются все данные БД (по умолчанию 50 и это настраивается).

**Reply by ttm-alexeypolunin** [reply]
Подумал, что криво объяснил: «в Селект не передаются все данные БД» — имеется в виду, что поиск ведется по всем, но на фронт в поле передается ограниченное количество.

**Reply by dmbrtlt** [reply]
Отлично, спасибо за ответ!
